src/hw_capemgr.js

```diff
@@ -9,9 +9,10 @@ var gpioFile = {};
 var pwmPrefix = {};
 var ainPrefix = "";
 
-exports.logfile = '/var/lib/cloud9/bonescript.log';
+module.exports = {
+logfile:  '/var/lib/cloud9/bonescript.log',
 
-exports.readPWMFreqAndValue = function (pin, pwm) {
+readPWMFreqAndValue:  function (pin, pwm) {
```

Comment:
If these are under curly braces, they should be indented. This would be a lot of indentation. Is there something wrong with simply replacing 'exports' with 'module.exports'? What will be the syntax now for local functions and variables?

Reply:
I don't think there is anything wrong with replacing exports with module.exports , for local functions it will be module.exports.readPWMFreqAndValue() if the export is done as above and no change if the module.exports is at the end of the file referencing the functions , i.e the functions are declared outside the module.exports block